config: NewSDK can return valid TracerProvider (#4741)
open-telemetry/opentelemetry-go-contrib: CHANGELOG.md

- Add the new `go.opentelemetry.io/contrib/instrgen` package to provide auto-generated source code instrumentation. (#3068, #3108)
- Add `SDK.Shutdown` method in `"go.opentelemetry.io/contrib/config"`. (#4583)

> - The `config.NewSDK` function now returns a configured SDK with a valid `TracerProvider`. (#4741)

Reviewer: ```suggestion
- `NewSDK` in `go.opentelemetry.io/contrib/config` now returns a configured SDK with a valid `TracerProvider`. (#4741)
```
